libs/shared/ui/src/components/avatar-modal/AvatarModal.tsx

```diff
@@ -0,0 +1,61 @@
+import { AvatarImages } from '../../assets/mocks/avatarImages';
+import { Modal } from '../modal/Modal';
+
+import { AvatarImage } from './AvatarModal.styled';
+
+interface AvatarModalProps {
+  avatarImages: AvatarImages[];
+  showModal: boolean;
+  setShowModal: (showModal: boolean) => void;
+  selectedId: number;
+  setSelectedId: (id: number) => void;
+  appliedId: number;
+  onApplyId: (id: number) => void;
+}
+
+export const AvatarModal = ({
+  avatarImages,
+  showModal,
+  setShowModal,
+  selectedId,
+  setSelectedId,
+  appliedId,
+  onApplyId,
+}: AvatarModalProps) => {
+  const handleApply = () => {
+    onApplyId(selectedId);
+    setShowModal(false);
```

Comment:
`setShowModal` comes from the parent setter function, when we have these types of callbacks we rather use the name e.g. `onShowModal`.
`setShowModal` should be called that way in cmp where has been defined.
So, just a naming issue.

Reply:
The same applies to `setSelectedId`.